.circleci/config.yml

```diff
@@ -280,22 +256,6 @@ workflows:
           name: Optional UI Tests (iPad)
           <<: *ipad_test_device
           requires: [ "Build Tests" ]
-  Installable Build:
-    unless:
-      or:
-        - << pipeline.parameters.beta_build >>
-        - << pipeline.parameters.release_build >>
```

Comment:
So, in theory we should also take the occasion to remove those 2 parameters where they are declared from the top of the pipeline (L9-L15) because I don't think they're used anywhere anymore…

But in practice since we'll soon get rid of the whole `.circleci/config.yml` file very soon… we don't really care 😛

Reply:
In this case we're removing installable builds, not release builds though – Buildkite produces them now, so there was confusion still having the option for CircleCI to produce them too.

Reply:
Oh so they're supposed to be telescoping / chained / related PRs, I see! (I usually reference the other PRs in the PR description when there's a dependency like that to avoid issues with merging them out of order and risking breaking stuff 😅 )

I see the 6144 targets the `release/*` branch while this one targets `trunk` (instead of either `release/*`, or being built on top of / targeting `add/release-builds`), is that intentional then?

Reply:
> In this case we're removing installable builds, not release builds though


🤦  I should stop reviewing PRs late in my day 😓 😛